common/src/main/java/tachyon/worker/WorkerNetAddress.java

```diff
@@ -40,6 +41,18 @@ public WorkerNetAddress(String host, int rpcPort, int dataPort, int webPort) {
   }
 
   /**
+   * Constructs the worker net address from thrift construct.
+   *
+   * @param netAddress the thrift net address
+   */
+  public WorkerNetAddress(NetAddress netAddress) {
```

Comment:
I think it is common for an address to consist of a hostname and a port, but not a hostname and multiple ports. But I am fine with WorkerNetAddress.